WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsPluginNotActivatedView.swift

```diff
@@ -23,23 +25,23 @@ struct InPersonPaymentsPluginNotActivated: View {
 
 private enum Localization {
     static let title = NSLocalizedString(
-        "Activate WooCommerce Payments",
-        comment: "Title for the error screen when WooCommerce Payments is installed but not activated"
+        "Activate %@",
```

Comment:
We should use positional placeholders: https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/localization.md

I am afraid this has to be updated before merging, otherwise the automated process that extract the strings for localization could fail

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5935/commits/94aaad6168c39dd525bc63b2dba938d888860b3e